lib/sequelizerc.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 const path = require('path');
+const fs = require('fs');
 
+const migrationsPath = path.resolve('migrations'),
+  modelsPath = path.resolve('app/model');
+if (!fs.existsSync(migrationsPath)) {
+  fs.mkdirSync(migrationsPath);
```

Comment:
please use `mkdirp` module to ensure mkdir won't fail if parent directory, and also avoid check if migration path exists.

```js
mkdirp.sync(migrationPath);
```